packages/sdk/src/sdk/api/challenges/types.ts

```diff
@@ -53,35 +53,28 @@ const DefaultSpecifier = z.object({
 const ReferralSpecifier = z.object({
   /** The challenge identifier. As in, the challenge "name." */
   challengeId: z.enum([ChallengeId.REFERRALS, ChallengeId.VERIFIED_REFERRALS]),
-  /** The user ID of the user that was referred. */
-  referredUserId: HashId,
   /** The user ID of the user completing the challenge. */
-  userId: HashId
-})
-
-const TrackSellerSpecifier = z.object({
-  /** The challenge identifier. As in, the challenge "name." */
-  challengeId: z.enum([ChallengeId.AUDIO_MATCHING_BUYER]),
-  /** The user ID of the owner of the track purchased. */
-  sellerUserId: HashId,
-  /** The track ID that was purchased. */
-  trackId: HashId
+  userId: HashId,
+  /** The user ID of the user that was referred. */
+  referredUserId: HashId
 })
 
-const TrackBuyerSpecifier = z.object({
+const AudiuMatchSpecifier = z.object({
```

Comment:
is this supposed to be `Audius` or `Audio`?

Reply:
Nice catch - Audio! Will fix